docker/Dockerfile.server

```
@@ -2,9 +2,24 @@
 
 FROM mcr.microsoft.com/windows/server:ltsc2022
 
+# OCI labels for metadata
+LABEL org.opencontainers.image.title="Windows Server LTSC 2022 Image pre-installed with uv and VC Redist"
+LABEL org.opencontainers.image.description="A Windows Server image with the uv python package and project manager from astral  installed, Python version 3.9-3.13 pre-installed, and including the Visual C++ Redistributable."
+LABEL org.opencontainers.image.version="0.0.2"
+# LABEL org.opencontainers.image.authors="Sebastian Weigand <s.weigand.phy@gmail.com>"
+LABEL org.opencontainers.image.url="https://github.com/s-weigand/uv-windows-docker"
+LABEL org.opencontainers.image.documentation="https://github.com/s-weigand/uv-windows-docker/blob/main/README.md"
+LABEL org.opencontainers.image.source=https://github.com/s-weigand/uv-windows-docker
+LABEL org.opencontainers.image.licenses="Apache-2.0"
 
 SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
 
+RUN $vcRedistUrl='https://aka.ms/vs/17/release/vc_redist.x64.exe'; `
+    $installerPath='C:\vc_redist.x64.exe'; `
+    Invoke-WebRequest -Uri $vcRedistUrl -OutFile $installerPath -UseBasicParsing; `
```

Let's use `WebClient.DownloadFile` since it is a lot faster for big files.
```suggestion
    (New-Object System.Net.WebClient).DownloadFile($vcRedistUrl, $installerPath); `
```

---

Reviewer suggestion adopted and incorporated in commit 087e34076735b42bedc516abebf11506294b2b42.